frontend_lib/test/fixture/workspace.js

```diff
@@ -0,0 +1,59 @@
+import { author } from './author.js'
+
+export const workspace = {
+  agenda_enabled: true,
+  created: '2020-06-16T14:12:33Z',
+  description: 'descripton',
+  is_deleted: false,
+  label: 'label',
+  owner: author,
+  public_download_enabled: true,
+  public_upload_enabled: true,
+  sidebar_entries: [
```

Comment:
put the sidebar entry list in a specific fixture file, import it in this file and change this line to be:
`sidebar_entries: fixtureSidebarEntryNameOfYourChoice`

It'll be useful to have only the sidebar entries fixture later

Reply:
Changed